libtelemetry/src/main/java/com/mapbox/services/android/telemetry/TelemetryClient.java

```diff
@@ -48,6 +49,14 @@ public void sendEvent(Event event, Callback callback) throws IOException {
     sendBatch(oneEvent, callback);
   }
 
+  void setDebugLoggingEnabled(boolean debugLoggingEnabled) {
```

Comment:
get/set nomenclature is hijacked by the Java language and should be only used when defining POJOs/DTOs.

In any case, this should be included as a setting of `TelemetryClientSettings`, per consistency with the rest of the settings.